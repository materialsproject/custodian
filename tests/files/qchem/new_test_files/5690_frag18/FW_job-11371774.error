/global/u2/s/sblau/pymatgen/pymatgen/io/cif.py:44: UserWarning: Please install optional dependency pybtex if youwant to extract references from CIF files.
  warnings.warn("Please install optional dependency pybtex if you"
DEBUG:launchpad:FW with id: 5690 is unique!
DEBUG:launchpad:Created/updated Launch with launch_id: 7682
DEBUG:launchpad:RUNNING FW with id: 5690
INFO:rocket.launcher:RUNNING fw_id: 5690 in directory: /scratch2/scratchdirs/sblau/ion_placement/block_2018-11-13-02-29-44-974177/launcher_2018-11-14-07-55-03-617059
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
INFO:rocket.launcher:Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
INFO:custodian.custodian:Run started at 2018-11-13 23:57:14.870338 in /scratch2/scratchdirs/sblau/ion_placement/block_2018-11-13-02-29-44-974177/launcher_2018-11-14-07-55-03-617059.
INFO:custodian.custodian:Custodian running on Python version 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 18:10:19)  [GCC 7.2.0]
INFO:custodian.custodian:Hostname: nid02700, Cluster: unknown
INFO:custodian.custodian:Starting job no. 1 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 2 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 3 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 4 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 5 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 6 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Logging to custodian.json...
INFO:custodian.custodian:Run ended at 2018-11-14 05:15:20.606685.
INFO:custodian.custodian:Run completed. Total time taken = 5:18:05.736347.
