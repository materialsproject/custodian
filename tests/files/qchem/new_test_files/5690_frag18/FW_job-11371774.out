foobar
2018-11-13 23:57:06,439 INFO Hostname/IP lookup (this will take a few seconds)
2018-11-13 23:57:06,441 INFO Launching Rocket
2018-11-13 23:57:14,833 INFO RUNNING fw_id: 5690 in directory: /scratch2/scratchdirs/sblau/ion_placement/block_2018-11-13-02-29-44-974177/launcher_2018-11-14-07-55-03-617059
2018-11-13 23:57:14,859 INFO Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
2018-11-13 23:57:14,867 INFO Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
2018-11-13 23:57:14,870 INFO Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
